sardana_adlink/ctrl/AdlinkAICoTiCtrl.py

```diff
@@ -232,15 +232,21 @@ def LoadOne(self, axis, value, repetitions, latency):
         self.AIDevice["NumOfTriggers"] = self._repetitions
         self.AIDevice['ChannelSamplesPerTrigger'] = chn_samp_per_trigger
 
+    def PreStartAll(self):
+        pass
+
     def PreStartOne(self, axis, value=None):
         if axis != 1:
             self._master_channel = axis
         return True
+    
+    def StartOne(self, axis, value=None):
```

Comment:
Sorry this is bug in Sardana that I introduced. It whould not be mandatory to implement it - see https://github.com/sardana-org/sardana/pull/1403.